### DISCUSSION TITLE
Impoet XLSX on Browser???

### DISCUSSION THREAD

**Discussion by danielmlc** [exceljs/exceljs#2506] [Q&A]
How to import execl on the front end?

**Comment by mah110020** [comment]
After some trouble, I now import exceljs as

`
import * as exceljs from "exceljs/dist/exceljs.min.js";
`

But hopefully in the future I will not need to specify the extended path.

**Comment by grbspltt** [comment]
`const Excel = require('exceljs/dist/es5/exceljs.browser');`

Regular exceljs import was looking for node server side dependencies.  If you are using a bundler like webpack you shouldn't import the minified (`.min.`) file.  It will automatically bundle and minifiy the files for you.
